### PR TITLE
fix: quiz edition

### PR DESCRIPTION
**Pull request by Guekka**
Ref: #299

### CODE REVIEW

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts [hunk 1]
```diff
@@ -78,46 +75,37 @@ export class QuestionEditGeneralComponent implements OnInit {
...
-              image: question.data.image,
-              type: question.data.type,
-            });
-            this.dialog.closeAll();
-          });
-      }
-    }
+    if (!this.form.valid) return;
+
+    const s = this.questionService;
```
Comment (Apoorva64) [inline apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts:96]: ? clean this

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.html [hunk 2]
```diff
@@ -54,16 +54,7 @@ <h3 class="card-title">Question</h3>
...
   <div class="card-footer">
-    <button
-      type="submit"
-      class="btn btn-primary"
-      [disabled]="loading"
-      (click)="onSubmit()"
-    >
-      <span
-        *ngIf="loading"
-        class="spinner-border spinner-border-sm mr-1"
```
Comment (Apoorva64) [inline apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.html:65]: why remove?

Comment (Guekka) [reply apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.html:65]: It wasn't used anywhere in the code

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts (hunk 1 above)
Comment (Guekka) [reply apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts]: Clean what?

**Comment by Guekka** [comment]
@Apoorva64 Please look at the image part. I did not find a way to get `dataURL` directly from the `FileFieldComponent`

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts (hunk 1 above)
Comment (Apoorva64) [reply apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts]: variable s

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.html (hunk 2 above)
Comment (Apoorva64) [reply apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.html:65]: use it then xD when subsctibing to post

File: apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts [hunk 3]
```diff
@@ -73,37 +73,63 @@ export class QuestionEditGeneralComponent implements OnInit {
...
+  get question(): Observable<Prisma.QuestionUpdateInput> {
+    return this.imageUrl.pipe(
+      map((url) => {
+        return {
+          title: this.formControls['title'].value as string,
+          image: url,
+          type: this.formControls['type'].value as QuestionType,
+          quiz: {
+            connect: {
+              id: this.quizId,
```
Comment (Apoorva64) [inline apps/front-end/back-office/src/app/components/quiz-creation/question-edit-general/question-edit-general.component.ts:104]: should not connect when updating

File: apps/front-end/front-office/src/app/components/quiz/quiz-item/quiz-item.component.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ export class QuizItemComponent {
   }
 
   get quizImageUrl(): string {
-    return this.quiz.imageUrl;
+    return this.quiz.imageUrl || '';
```
Comment (Apoorva64) [inline apps/front-end/front-office/src/app/components/quiz/quiz-item/quiz-item.component.ts:28]: no should be or default img

**Review by Apoorva64 — APPROVED** [review]

**Pull request merged**